### ISSUE TITLE
Split into two mods in Minecraft 1.17

### ISSUE THREAD

**Issue by SilentChaos512** [SilentChaos512/SilentGems#511]
## Versions

- Minecraft 1.17.x

## What Should Be Added or Changed?

The mod should be split into two separate mods: one that focuses on world generation and basic items ("Silent's Gems" or maybe "Silent's Gems: Base"), and another ("Silent's Gems: Chaos") that contains the chaos mechanics and more overpowered/unstable blocks and items, like the supercharger and teleporters.

The base of Silent's Gems could possibly add a new, less overpowered version of the supercharger. Then the "Chaos" add-on mod could keep the old supercharger.

### Silent's Gems: Base

- Gems: items, ~shards~, ores ~(including multi-ores)~, blocks, bricks, glass, lamps
- Glowroses and fertilizer
- Pedestals (new, basic type that doesn't work with chaos gems, maybe?)
- Starlight Charger (new block, weaker version of the supercharger) _(edit: maybe add to Silent Gear instead?)_
- Gem bag and glowrose basket
- Foods
- Gear materials
- Pet summoners

### Silent's Gems: Chaos

- Teleporters, including return home charms, anchors, and linker
  - Reasoning: Teleporters create chaos and the teleportation code usually breaks completely every major MC update, taking awhile to fix.
- Chaos gems and runes
- Chaos pedestals (new blocks, replacing old pedestals)
- Transmutation altar
- Purifier
- Supercharger
- Luminous flower pot and phantom lights
  - Reasoning: Pot is buggy, weird and often inconvenient light placement. And I just don't like this block anymore.
- Chaos slimes and wisps
- Chaos meter
- Corrupting and purifying powder
- Chaos orbs
- Most enchantments and potion effects

### Move to Silent Gear

- Fluffy blocks/puffs/plants/etc.

### Unsure

Feedback welcome for these. ~~I feel like all the **soul stuff** would fall into the category of "buggier" features, so they might be better suited for Chaos instead of Base.~~

The **token enchanter** and **enchantment tokens** have been moved to a new mod.

- ~~Pet summoners~~ (add to Base)
- ~~Token enchanter and enchantment tokens~~ -- Moved to a new mod: [Token Enchanter](https://github.com/SilentChaos512/Token-Enchanter)
- Soul gems (probably Base)
- ~~Soul urns~~ -- Replaced by [Loginar Storage](https://www.curseforge.com/minecraft/mc-mods/loginar-storage)
- Gear souls

## Why Should This Be Done?

The chaos mechanics are not particularly popular, and can be confusing to new players. The mod also adds way too much for a single mod, so many pass on Silent's Gems because they just want the ores and basic items.

**Comment by Funnymeme1** [comment]
It feels like the starlight charger would be a better fit for Silent Gear. After all, basically every material in basically every mod/datapack I've seen has a Chargeability stat, and it's a bit of a shame that you have to use an entirely different mod just to use it.

**Comment by SilentChaos512** [comment]
That's fair, probably would be best to add it to Gear instead. It's still just a rough idea at this point, of course.

**Comment by unmoving-javi** [comment]
Personal opinion here, but I really like Gear Souls. The soul mechanics (in my opinion) would be a good fit for Silent Gear with some tweaking. Soul gems admittedly have some weird issues related to them, but that could maybe be fixed with Skyrim-style soul gems (i.e. filled by killing a mob with an empty one in your inventory?). That would also reduce the randomness somewhat so you could get a gear soul with the aspects you want, though I'm not sure how the balance on that works exactly.
In any case I think the soul mechanics are nice and I'd be sad to see them dropped entirely.
As for soul urns, I have no idea why they require soul gems. Those should probably be reworked at some point.

**Comment by AbsorbingSafe11** [comment]
Would it bee too much work to make the chaos stuff part of Silent's Mechanisms, as a part of that mod's late game?
(or use energy instead of causing chaos for any of the chaos stuff)
I ask because, while I like Silent's Mechanisms, it lacks an endgame set of machines or technological goal for players to work towards and stuff like teleporters, transmutation, and even phantom lights fit well for and endgame goal of a tech mod.
Also, while unrelated to the previous statement,  can the pet summoning stuff be put into the future Silent's Gems; Base as most players (I would think) like the convenience of easily summoned pets.

**Comment by DiamondChicky36** [comment]
I suggest that in 1.17, you should keep the amethyst ores, blocks and etc, however, rename it to 'Amethyst Gem' to avoid confusion, and give it a more purple-magenta tint. Also I recommend the ametrine be recolored

**Comment by SilentChaos512** [comment]
@DiamondChicky36 See #513, it's already been decided to remove amethyst, along with many other gems.

**Comment by DiamondChicky36** [comment]
If that's the case, how about gem shards? Maybe more decorative blocks could be made.
(not a lot)

**Comment by SilentChaos512** [comment]
I decided to remove shards for now since they felt unnecessary. Bricks and glass blocks had their recipes changed to use full gems and give more blocks per crafting.

**Comment by DiamondChicky36** [comment]
I have another idea, more types of quartz, maybe fictional ones? I'm not
really sure. Crystals are my kinda thing

**Comment by CoolDemon96** [comment]
Could you move gear souls to a separate mod.

**Comment by sophienix** [comment]
could add stats back to apatite imperial IV since thermal have it this way people can still 8x diamonds when they stack it with fortune same with aquamarine add water breathing back to it since astral sorcery adds that

**Comment by Splarkszter** [comment]
This is very confusing to me. What is the version that is meant to be played?, can both mods co-exist?

**Comment by SilentChaos512** [comment]
Yes, although the Chaos mod was never really finished and barely adds anything.

**Comment by Splarkszter** [comment]
Oh, my bad, i thought this a was Silent Gems/Silent Gear post, didn't read well... 
Sooo, my question still the same, but about SGems and SGear.

**Comment by SilentChaos512** [comment]
The answer is still the same, although Silent's Gems has significantly less utility without Silent Gear. If you want tools and armor, you need Gear. But both mods can run separately.

**Comment by DaddyKakashi** [comment]
Personally would love to see soul gems and soul urns both in base.
They're basically another form of shulkers with alot more utility and customization options.
Maybe I'm just bias because I use them so much in versions with them lol

**Comment by SilentChaos512** [comment]
The urns actually ended up becoming a new mod recently, [Loginar Storage](https://www.curseforge.com/minecraft/mc-mods/loginar-storage). But soul gems are still a work in progress...

**Comment by DaddyKakashi** [comment]
Oh wasn't even aware of that particular mod for newer releases. Thank you so much for the link and for making such an amazing mod ^-^

**Comment by AnimationJack** [comment]
I was just curious if you were going to at least add the teleporters and the return home charms to the chaos mod, If not that's fine. Preferably 1.19.2
